robot/util/utils.py

```diff
@@ -36,3 +36,8 @@ def get_pid(keyword):
 
     return -1
 
+def write_to_file(filename, text, locationx, locationy):
```

Comment:
You're writing a high level function that is supposed to be abstract like "Write To File" but it takes arguments about the small details. I would try not to pass locationx and locationy, why do you need that? Did you just want to append?

Reply:
Good point, removed arguments `locationx` and `locationy` and renamed function to `append_to_file`.